assets/js/blocks/attribute-filter/style.scss

```diff
@@ -67,3 +67,10 @@
 		}
 	}
 }
+
+.editor-styles-wrapper  .wc-block-attribute-filter__button.wc-block-attribute-filter__button,
+.wc-block-attribute-filter__button.wc-block-attribute-filter__button {
+	padding: $gap-smaller $gap;
+	font-size: 0.875em;
```

Comment:
Nit: you can use the [`font-size( small )` mixin](https://github.com/woocommerce/woocommerce-blocks/blob/d8eba08ed53787caf00a071a08a4023b6c89b5cb/assets/css/abstracts/_mixins.scss#L1-L13) here. :slightly_smiling_face: 

This is useful to make sure we never use a font-size different from the five defined.